src/cephadm/containers/keepalived/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN microdnf install --nodocs \
+    bash       \
+    curl       \
+    iproute    \
+    keepalived-2.1.5 \
```

Comment:
I verified these are all in Red Hat's ubi8 repos, so we do not need CentOS's repos. That's great.